packages/eas-cli/src/build/utils/printBuildInfo.ts

```diff
@@ -82,10 +84,14 @@ function printBuildResult(accountName: string, build: BuildFragment): void {
       buildId: build.id,
       account: accountName,
     });
+    const installUrl = getInstallUrl(build);
+    if (installUrl) {
```

Comment:
`build.artifacts?.buildUrl` is never false when build status is `Finished`, so maybe assert here would be a better fit.
either on installUrl or inside `getInstallUrl` function